Ghidra/Features/BytePatterns/src/main/java/ghidra/app/analyzers/FunctionStartAnalyzer.java

```diff
@@ -723,9 +725,8 @@ public boolean added(Program addedProgram, AddressSetView addedSet,
 						while (referencesTo.hasNext()) {
 							Reference reference = referencesTo.next();
 							if (reference.getReferenceType().isConditional()) {
-								continue;
+								continue outerloop;
```

Comment:
Our convention would be to set a flag, rather than use a goto.

Reply:
Fixed!